ui/raidboss/emulator/data/NetworkLogConverter.js

```diff
@@ -17,9 +11,9 @@ export default class NetworkLogConverter {
     return ret;
   }
 
-  async convertLines(lines) {
+  async convertLines(lines, repo) {
     this.Combatants = {};
-    const repo = new LogRepository();
+    repo = repo || new LogRepository();
```

Comment:
To make this more readable to me, can the one other caller pass in a new repo too, rather than having this be conditionally implicit?

Reply:
Yeah, I can adjust the other caller to pass in `repo` as well. This was my aversion to passing around large objects in other languages coming back to bite me 😅.

Reply:
Can this line be removed, then?